draft-ietf-rtcweb-jsep.xml

```diff
@@ -793,7 +793,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           target="I-D.ietf-mmusic-sdp-bundle-negotiation"></xref>.
           Regardless of policy, the application will always try to negotiate BUNDLE
           onto a single transport,
-          and will offer a single BUNDLE group across all media sections.
+          and will offer a single BUNDLE group across all media sections. The
```

Comment:
Given the text below that talks about non-BUNDLE-aware endpoints, I think this sentence would be best written as
"... and will offer a single BUNDLE group across all media sections; use of this single transport is contingent upon the answerer accepting BUNDLE."